handler/src/main/java/io/netty/handler/ssl/ReferenceCountedOpenSslServerContext.java

```diff
@@ -178,6 +179,14 @@ static OpenSslServerSessionContext newSessionContext(ReferenceCountedOpenSslCont
 
             OpenSslServerSessionContext sessionContext = new OpenSslServerSessionContext(thiz, keyMaterialProvider);
             sessionContext.setSessionIdContext(ID);
+            // Enable session caching by default
+            sessionContext.setSessionCacheEnabled(true);
+            if (sessionCacheSize > 0) {
+                sessionContext.setSessionCacheSize((int) Math.min(sessionCacheSize, Integer.MAX_VALUE));
+            }
+            if (sessionTimeout > 0) {
```

Comment:
1. Same comment as for the client:
> Why only positive numbers? Zero is also a legit value.
Negative values should throw an `IllegalArgumetException`, consider verifying input parameters from public API of `SslContextBuilder` or `OpenSslClientContext` ctors.

2. Does that mean that we have an external cache for the server side as well?

Reply:
yes... we use the same "mechanism" for client and server.